practicas/EARSV/P2/tspln_práctica2.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+"""TSPLN_Práctica2.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1AIMj8DkK764oL7eAtlOiG_XwZgLJOKcp
+
+**Alumno:** Romero Sánchez del Valle Efraín Alejandro
+"""
+
+!pip install spacy datasets matplotlib
+!python -m spacy download es_core_news_md
+
+import random
+import string
+from collections import Counter
+
+import spacy
+from collections import Counter
+from datasets import load_dataset
+import matplotlib.pyplot as plt
+
+"""#Problema 1"""
+
+# Parámetros
+num_palabras = 10000  # Número total de palabras generadas
+longitud_palabra_max = 10  # Longitud máxima de las palabras
+palabras=[]
```

Comment:
Recuerda seguir las buenas prácticas de estilo de python
```suggestion
palabras = []
```